src/extension.ts

```diff
@@ -261,11 +257,9 @@ async function modulesCallersCommand(languageClient: TerraformLanguageClient, mo
   return execWorkspaceCommand(languageClient.client, requestParams);
 }
 
-async function moduleCallers(
-  languageClient: TerraformLanguageClient,
-  moduleUri: string,
-): Promise<moduleCallersResponse> {
-  const response = await modulesCallersCommand(languageClient, moduleUri);
+export async function moduleCallers(moduleUri: string): Promise<moduleCallersResponse> {
+  const client = clientHandler.getClient();
```

Comment:
We should check for an undefined client here

Reply:
Good catch! I've added a check.